### PR TITLE
Haptic feedback

### LINKED ISSUE #1111: [FEATURE] Haptic feedback
Add a small vibration feedback for the following events:
- When a player deals or receives damage.
- When a player cancels a skill.

### PR DESCRIPTION
**Pull request by AngieDutra**
Closes #1111 

## Motivation

The game is currently missing vibration feedback

## Summary of changes

Haptic feedback was added when the skill is canceled and when a player receives damage. To test this the reviewer must build this and test it on their phone

## Attention
I'm aware the issue involves applying the feedback for when it damages other players but we don't have that data from backend yet. For now we only know when the damage is applied to us but not who we attack. When that information is available the vibration can be added.

## Checklist
- [x] I have tested the changes locally.
- [x] I self-reviewed the changes on GitHub, line by line.
- [ ] Tests have been added/updated.
- [ ] This change requires new documentation.
  - [ ] Documentation has been added/updated.
- [x] I have tested the changes in another devices.
  - [ ] Tested in iOS.
  - [x] Tested in Android.

### CODE REVIEW

File: client/Assets/Scripts/CharacterFeedbacks.cs [hunk 1]
```diff
@@ -90,6 +88,18 @@ public void ChangePlayerTextureOnDamage(float clientHealth, float playerHealth)
         }
     }
 
+    public void HapticFeedbackOnDamage(float clientHealth, float playerHealth, ulong playerId)
+    {
+        if (
+            clientHealth != playerHealth
```
Comment (tkz00) [inline client/Assets/Scripts/CharacterFeedbacks.cs:94]: This condition is redundant with the previous one, the one on line 94 should be removed.

**Review by tkz00 — CHANGES_REQUESTED** [review]

File: client/Assets/Scripts/CharacterFeedbacks.cs (hunk 1 above)
Comment (AngieDutra) [reply client/Assets/Scripts/CharacterFeedbacks.cs:94]: I agree. Good catch. Done ✨

**Review by tkz00 — APPROVED** [review]
Looks good!

**Comment by BertovDev** [comment]
Nice! Working so good. It is possible to add more vibration when you are in the danger zone ? So it will feel like you are trully in danger.

**Comment by AngieDutra** [comment]
> It is possible to add more vibration when you are in the danger zone ?

Sure! I believe it should be easy enough and it is within the scope of the issue since it involves damage feedback

**Pull request merged**